src/Authentication/logincards/passwordresetemailcard/PasswordResetEmailCard.js

```diff
@@ -69,7 +69,7 @@ function PasswordResetEmailCard({ onSubmit, backToLogin }) {
       await firebaseApp.auth().sendPasswordResetEmail(values.email);
       onSubmit();
     } catch (e) {
-      console.log(e);
+      //console.error(e);
```

Comment:
Not sure if you wanted to comment this out, but I'll just point it out so you are aware.

Reply:
Yeup. It is frontend so we don't need that displaying. But it is a commonly used error that we should log in the future.

Reply:
Wanted to keep it commented so i could find all the console.errors later for the frontend